sdk/batch/azure-compute-batch/src/main/java/com/azure/compute/batch/auth/BatchSharedKeyCredentialsPolicy.java

```diff
@@ -49,7 +49,7 @@ private synchronized String sign(String stringToSign) {
             // Encoding the Signature
             // Signature=Base64(HMAC-SHA256(UTF8(StringToSign)))
             byte[] digest = getHmac256().doFinal(stringToSign.getBytes("UTF-8"));
-            return Base64.encodeBase64String(digest);
```

Comment:
It looks like these aren't entirely equivalent, but I'm not sure which one is right.

If my reading is correct, the Apache commons versions does MIME encoding (max line length of 76, \r\n line separators) while Java's `getEncoder()` does plain base 64 encoding and doesn't have a max line length or add any carriage returns or line feeds. Java does have `getMimeEncoder()` which sounds like it behaves like the Apache Commons version, though.

Assuming I'm correct that they're not the same, do you know which is the correct way of doing Base64 encoding for shared keys?

Reply:
Looks like the server-side base64 decoder will handle either MIME or non-MIME base64 strings, so as long as tests pass this shouldn't be an issue. I'll approve the PR.